Looks like you included some Python 3.6 only code in your requirement 

homeassistant/components/geniushub/__init__.py:7:0: E0001: Cannot import 'geniushubclient' due to syntax error 'Variable annotation syntax is only supported in Python 3.6 and greater (<unknown>, line 568)' (syntax-error)

--- a/homeassistant/components/geniushub/__init__.py
+++ b/homeassistant/components/geniushub/__init__.py
@@ -54,6 +54,9 @@ async def async_setup(hass, hass_config):
             exc_info=True)
         return False
 
+    _LOGGER.debug("zones_raw = %s", data._client.hub._zones_raw)
+    _LOGGER.debug("devices_raw = %s", data._client.hub._devices_raw)
+
     async_track_time_interval(hass, data.async_update, SCAN_INTERVAL)
 
     for platform in ['climate', 'water_heater']:
@@ -84,4 +87,9 @@ class GeniusData:
         except AssertionError:  # assert response.status == HTTP_OK
             _LOGGER.warning("Update failed.", exc_info=True)
             return
+
+        _LOGGER.debug("zones_raw = %s", self._client.hub._zones_raw)
+        _LOGGER.debug("devices_raw = %s", self._client.hub._devices_raw)
+
         async_dispatcher_send(self._hass, DOMAIN)
+

--- a/homeassistant/components/geniushub/__init__.py
+++ b/homeassistant/components/geniushub/__init__.py
@@ -54,8 +54,8 @@ async def async_setup(hass, hass_config):
             exc_info=True)
         return False
 
-    _LOGGER.debug("zones_raw = %s", data._client.hub._zones_raw)
-    _LOGGER.debug("devices_raw = %s", data._client.hub._devices_raw)
+    _LOGGER.debug("zones_raw = %s", data._client.hub._zones_raw)  # noqa; pylint: disable=protected-access
+    _LOGGER.debug("devices_raw = %s", data._client.hub._devices_raw)  # noqa; pylint: disable=protected-access
 
     async_track_time_interval(hass, data.async_update, SCAN_INTERVAL)
 
@@ -88,8 +88,7 @@ class GeniusData:
             _LOGGER.warning("Update failed.", exc_info=True)
             return
 
-        _LOGGER.debug("zones_raw = %s", self._client.hub._zones_raw)
-        _LOGGER.debug("devices_raw = %s", self._client.hub._devices_raw)
+        _LOGGER.debug("zones_raw = %s", self._client.hub._zones_raw)  # noqa; pylint: disable=protected-access
+        _LOGGER.debug("devices_raw = %s", self._client.hub._devices_raw)  # noqa; pylint: disable=protected-access
 
         async_dispatcher_send(self._hass, DOMAIN)
-

--- a/homeassistant/components/geniushub/manifest.json
+++ b/homeassistant/components/geniushub/manifest.json
@@ -3,7 +3,7 @@
   "name": "Genius Hub",
   "documentation": "https://www.home-assistant.io/components/geniushub",
   "requirements": [
-    "geniushub-client==0.4.13"
+    "geniushub-client==0.4.14"
   ],
   "dependencies": [],
   "codeowners": ["@zxdavb"]

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -511,7 +511,7 @@ gearbest_parser==1.0.7
 geizhals==0.0.9
 
 # homeassistant.components.geniushub
-geniushub-client==0.4.13
+geniushub-client==0.4.14
 
 # homeassistant.components.geo_json_events
 # homeassistant.components.nsw_rural_fire_service_feed

